kratos/geometries/hexahedra_interface_3d_8.h

```diff
@@ -700,15 +700,15 @@ template<class TPointType> class HexahedraInterface3D8 : public Geometry<TPointT
     }
 
     /**
-     * Returns the local coordinates of a given arbitrary point
```

Comment:
The correct name of this commit is Adding const to interface geometries, but I got confused :P